esrfconcert/devices/motors/micos.py

```diff
@@ -1,10 +1,11 @@
 # TODO: do we still need this?
```

Comment:
@edobN, here I poke myself @tfarago to remind me to remove this comment before mering.